app/globals.css

```
@@ -3,20 +3,14 @@
 :root {
   --background: #ffffff;
   --foreground: #171717;
+  --drift: 14px;
+  color-scheme: light;
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)